package.json

```diff
@@ -47,7 +50,10 @@
     "@oclif/plugin-help": "^6.2.32",
     "@oclif/plugin-update": "^4.7.8",
     "graphql": "^16.11.0",
+    "inquirer": "^12.11.0",
```

Comment:
Inquirer is legacy
https://www.npmjs.com/package/inquirer

Node.js has builtin readline and some methods to prompt questions to users https://nodejs.org/api/readline.html